templates/user/settings.html

```diff
@@ -626,6 +630,29 @@ <h2 class="card-header" id="headingFour">
                             </div>
                           </div>
                         </div>
+                        <div class="card" style="box-shadow: none; ">
+                          <h2 class="card-header" id="headingOne">
```

Comment:
@manuelsc Thanks for the update. Not sure how relevant this is for the id here but `headingOne` is the same as for Lighthouse

Reply:
Good catch I'll raise a ticket internally to clean this up too. Seems to only have a descriptive impact